### DISCUSSION TITLE
Port request for Visual studio 2022

### DISCUSSION THREAD

**Discussion by sddania** [daltonmenezes/aura-theme#121] [Port request]
Port request for Visual studio 2022!

**Comment by catalin-ifacts** [comment]
Make a theme for Visual Studio (not Code!) and one for Notepad++

**Comment by catalin-ifacts** [comment]
@sddania Just found this, but I am too lazy, maybe you are not and you will upload the vssettings here, thanks :D
https://github.com/microsoft/theme-converter-for-vs

**Comment by sddania** [comment]
The theme now should be a .vsix extension on VS 2022; if i have time i will try to generate it.

**Comment by daltonmenezes** [comment]
Hi, I tried to make this port using this official converter during the v2.0.0 development, but the results aren't good and need a lot of work to make this acceptable. I'm not a Visual Studio user, so I need help. The Visual Studio Code port must be used as the guide to put the colors in a right way. If you would like to make it, just let me know to assignee you to this issue and let me know if you need some help. 💜
